src/components/interactive-builder/modals/question/question-form/question-types/inputs/test-order/test-order-type-question.test.tsx

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import TestOrderTypeQuestion from './test-order-type-question.component';
+import { FormFieldProvider } from '../../../../form-field-context';
+import type { FormField } from '@openmrs/esm-form-engine-lib';
+
+// Mock the ConceptSearch component
+jest.mock('../../../common/concept-search/concept-search.component', () => {
+  return function MockConceptSearch({
+    defaultConcept,
+    onSelectConcept,
+  }: {
+    defaultConcept?: string;
+    onSelectConcept: (concept: { uuid: string; name: string }) => void;
+  }) {
+    return (
+      <input
+        data-testid="concept-search"
+        defaultValue={defaultConcept}
+        onChange={(e) => onSelectConcept({ uuid: e.target.value, name: e.target.value })}
+        placeholder="Search for a concept"
+      />
+    );
+  };
+});
+
+const mockSetFormField = jest.fn();
+const formField: FormField = {
+  id: 'test-order-1',
+  type: 'testOrder',
+  questionOptions: {
+    rendering: 'repeating',
+    concept: '',
+    selectableOrders: [],
+  },
+};
+
+jest.mock('../../../../form-field-context', () => ({
+  ...jest.requireActual('../../../../form-field-context'),
+  useFormField: () => ({ formField, setFormField: mockSetFormField }),
+}));
+
+describe('TestOrderTypeQuestion', () => {
+  it('renders the component correctly', () => {
+    renderTestOrderComponent();
+
+    expect(screen.getByText('Concept')).toBeInTheDocument();
+    expect(screen.getByText('Selectable Orders')).toBeInTheDocument();
```

Comment:
Prefer regex to look for test, please update the other strings too
```suggestion
    expect(screen.getByText(/selectable orders/i)).toBeInTheDocument();
```